Automattic/jetpack
# VideoPress: Add chapters validation function

Part of #28588

## Proposed changes:

* Adds a chapters validation function, but does not use it yet.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Locally, `jetpack test packages/videopress js`
* Go to the block editor and add a VideoPress block
* Add some chapters in the description
* Check that the functionality still works as normal
